core/server/master/src/test/java/alluxio/master/AlluxioMasterRestServiceHandlerTest.java

```diff
@@ -91,9 +90,14 @@
   private BlockMaster mBlockMaster;
   private MasterRegistry mRegistry;
   private AlluxioMasterRestServiceHandler mHandler;
+
   @Rule
   public TemporaryFolder mTestFolder = new TemporaryFolder();
 
+  @Rule
+  public ConfigurationRule mConfigurationRule = new ConfigurationRule(
+      new ImmutableMap.Builder().put(PropertyKey.MASTER_MOUNT_TABLE_ROOT_UFS, TEST_PATH).build());
```

Comment:
it works in Java 7, it does the same thing as the 1-property constructor in `ConfigurationRule`